app/services/hyrax/collection_types/create_service.rb

```diff
@@ -11,13 +11,12 @@ module CollectionTypes
     # @see Hyrax:CollectionType
     #
     class CreateService
-      DEFAULT_MACHINE_ID = Hyrax::CollectionType::USER_COLLECTION_MACHINE_ID
-      DEFAULT_TITLE = Hyrax::CollectionType::USER_COLLECTION_DEFAULT_TITLE
       DEFAULT_OPTIONS = {
-        description: 'A User Collection can be created by any user to organize their works.',
+        description: '',
```

Comment:
Do we still need this hash entry?

Reply:
When you go through the UI, params has description as...
```
"description"=>""
```
If description is not included in the options passed in, then it will be nil.  The DEFAULT_OPTIONS provides an empty string as the default description.